### PR TITLE
Update muon script for running with old data

### PR DESCRIPTION
**Pull request by FrancaCassol**

### CODE REVIEW

File: lstchain/scripts/lstchain_data_muon_analysis.py [hunk 1]
```diff
@@ -62,6 +55,11 @@
                                          "Default = 442",
                     type = int, default = 442)
 
+parser.add_argument("--tel_id", help = "telescope id"
+                                         "Default = 1",
+                    type = int, default = 1)
```
Comment (rlopezcoto) [inline lstchain/scripts/lstchain_data_muon_analysis.py]: `tel_id` should be 0 for run number < xxx, right?
We should either specify the run number and select the tel_id with it or specify the tel_id without mentioning the run number

File: lstchain/scripts/lstchain_data_muon_analysis.py [hunk 2]
```diff
@@ -74,7 +72,7 @@ def main():
     print("max events: {}".format(args.max_events))
 
     # Camera geometry
-    geom = CameraGeometry.from_name("LSTCam-002")
+    geom = CameraGeometry.from_name("LSTCam-003")
```
Comment (rlopezcoto) [inline lstchain/scripts/lstchain_data_muon_analysis.py]: new camera geometry?

File: lstchain/scripts/lstchain_data_muon_analysis.py [hunk 3]
```diff
@@ -91,78 +89,57 @@ def main():
...
-    cal_data =  WaveformCalibrationContainer()
-    ped_data =  PedestalContainer()
+        r1_sample_start=2,r1_sample_end=38, tel_id=tel_id)
 
-    dc_to_pe = []
-    ped_median = []
+    charge_config = Config({
+        "LocalPeakWindowSum": {
+            "window_shift": 4,
+            "window_width": 8
```
Comment (rlopezcoto) [inline lstchain/scripts/lstchain_data_muon_analysis.py]: why using here different parameters as those used by default in the r0_to_dl1 analysis?

Comment (vuillaut) [reply lstchain/scripts/lstchain_data_muon_analysis.py]: Linked to #208

File: lstchain/scripts/lstchain_data_muon_analysis.py (hunk 1 above)
Comment (FrancaCassol) [reply lstchain/scripts/lstchain_data_muon_analysis.py]: Yes, indeed, what do you prefer?

File: lstchain/scripts/lstchain_data_muon_analysis.py (hunk 2 above)
Comment (FrancaCassol) [reply lstchain/scripts/lstchain_data_muon_analysis.py]: this should be the correct one see 
[https://github.com/cta-observatory/ctapipe-extra/pull/25](https://github.com/cta-observatory/ctapipe-extra/pull/25)
I am going to change it also in ctapipe_io_lst

File: lstchain/scripts/lstchain_data_muon_analysis.py (hunk 3 above)
Comment (FrancaCassol) [reply lstchain/scripts/lstchain_data_muon_analysis.py]: Sorry, actually I think I changed them looking at the waveforms (they seemed the best one).
We should indeed used the values inside a configuration file. Which value are used in the r0_to_dl1 (where are the configuration files?)
But the way, are muons pulses as shower pulses?

Comment (vuillaut) [reply lstchain/scripts/lstchain_data_muon_analysis.py]: Hi Franca.
Maybe we can take this opportunity to work on the merging of config files (in another PR) to get a single one?
The one used for `r0_to_dl1` is there: https://github.com/cta-observatory/cta-lstchain/blob/master/lstchain/data/lstchain_standard_config.json

Comment (FrancaCassol) [reply lstchain/scripts/lstchain_data_muon_analysis.py]: Hi @vuillaut,
ok, I will have a look, but I am going before to insert the time pulse correction in the calibration chain. Then, let do a bit of cleaning also in this part.

**Comment by FrancaCassol** [comment]
The content of this PR has moved to [https://github.com/cta-observatory/cta-lstchain/pull/263](https://github.com/cta-observatory/cta-lstchain/pull/263)

**Pull request closed without merging**